spec/middleware/trade_tariff_frontend/basic_auth_spec.rb

```diff
@@ -0,0 +1,65 @@
+class MockRackApp
+  attr_reader :request_body
+
+  def initialize
+    @request_headers = {}
+  end
+
+  def call(env)
+    @env = env
+    @request_body = env['rack.input'].read
+    [200, { 'Content-Type' => 'text/plain' }, %w[OK]]
+  end
+
+  def [](key)
+    @env[key]
+  end
+end
```

Comment:
should we move this mock inside 
`ControllerSpecHelper`
or
`RequestSpecHelper`
? 🤔